src/Sylius/Bundle/ShopBundle/Twig/Component/Account/Address/FormComponent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Twig\Component\Account\Address;
+
+use Sylius\Component\Core\Model\Address;
+use Sylius\TwigHooks\LiveComponent\HookableLiveComponentTrait;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\ComponentWithFormTrait;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+
+#[AsLiveComponent]
+class FormComponent
+{
+    use ComponentWithFormTrait;
+    use DefaultActionTrait;
+    use HookableLiveComponentTrait;
+
+    public ?Address $address = null;
+
+    /** @param class-string $formClass */
+    public function __construct(
+        private readonly FormFactoryInterface $formFactory,
+        private readonly string $formClass,
+    ) {
+    }
+
+    protected function instantiateForm(): FormInterface
+    {
+        return $this->formFactory->create($this->formClass, $this->address);
```

Comment:
```suggestion
        return $this->formFactory->create($this->formClass, $this->address, ['add_build_address_form_subscriber' => false]);
```
It should be handled the same way as the AdminBundle and also needs to be added to the route configuration